.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        php: [8.0, 8.1]
+        php: [8.0, 8.1, 8.2, 8.3]
```

Comment:
please remove support for PHP 8.0 as well.

Reply:
done